### PR TITLE
Adds autoscaling and CCR API request and response examples

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

Related to https://github.com/elastic/elasticsearch-specification/issues/2482.
This PR adds request and response examples to the autoscaling and CCR APIs.

### CODE REVIEW

File: specification/autoscaling/get_autoscaling_capacity/autoscalingApisGetAutoscalingCapacityResponseExample1.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "summary": "A successful respons for retrieving the current autoscaling capacity.",
```
Comment (lcawl) [inline specification/autoscaling/get_autoscaling_capacity/autoscalingApisGetAutoscalingCapacityResponseExample1.json:2]:
```suggestion
  "summary": "A successful response for retrieving the current autoscaling capacity.",
```

File: specification/autoscaling/get_autoscaling_policy/autoscalingApisGetAutoscalingPolicyResponseExample1.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "summary": "A successful respons for retrieving an autoscaling policy.",
```
Comment (lcawl) [inline specification/autoscaling/get_autoscaling_policy/autoscalingApisGetAutoscalingPolicyResponseExample1.json:2]:
```suggestion
  "summary": "A successful response for retrieving an autoscaling policy.",
```

File: specification/autoscaling/put_autoscaling_policy/autoscalingApisPutAutoscalingPolicyRequestExample2.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "summary": "Creates an autoscaling policy named `my_autoscaling_policy` using the fixed autoscaling decider, applying to the set of nodes having (only) the `data_hot` role",
```
Comment (lcawl) [inline specification/autoscaling/put_autoscaling_policy/autoscalingApisPutAutoscalingPolicyRequestExample2.json:2]: >  "summary": "Creates an autoscaling policy named `my_autoscaling_policy` using the fixed autoscaling decider, applying to the set of nodes having (only) the `data_hot` role",

... not a blocker but I think the summary ought to be shorter and this longer detail could move to the description.

File: specification/ccr/delete_auto_follow_pattern/ccrApisAutoFollowDeleteAutoFollowPatternResponseExample1.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "summary": "A successful respons for deleting an auto-follow pattern.",
```
Comment (lcawl) [inline specification/ccr/delete_auto_follow_pattern/ccrApisAutoFollowDeleteAutoFollowPatternResponseExample1.json:2]:
```suggestion
  "summary": "A successful response for deleting an auto-follow pattern.",
```

File: specification/ccr/pause_auto_follow_pattern/ccrApisAutoFollowPauseAutoFollowPatternResponseExample1.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "summary": "A successful respons for pausing an auto-follow pattern.",
```
Comment (lcawl) [inline specification/ccr/pause_auto_follow_pattern/ccrApisAutoFollowPauseAutoFollowPatternResponseExample1.json:2]:
```suggestion
  "summary": "A successful response for pausing an auto-follow pattern.",
```

File: specification/ccr/resume_auto_follow_pattern/ccrApisAutoFollowResumeAutoFollowPatternResponseExample1.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+  "summary": "A successful respons for resuming an auto-follow pattern.",
```
Comment (lcawl) [inline specification/ccr/resume_auto_follow_pattern/ccrApisAutoFollowResumeAutoFollowPatternResponseExample1.json:2]:
```suggestion
  "summary": "A successful response for resuming an auto-follow pattern.",
```

**Review by lcawl — APPROVED** [review]
Minor comments, otherwise LGTM

File: specification/autoscaling/put_autoscaling_policy/autoscalingApisPutAutoscalingPolicyRequestExample2.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+  "method_request": "PUT /_autoscaling/policy/my_autoscaling_policy",
+  "description": "The API method and path for this request: `PUT /_autoscaling/policy/my_autoscaling_policy`",
+  "type": "request",
+  "value": "{\n  \"roles\" : [ \"data_hot\" ],\n  \"deciders\": {\n    \"fixed\": {\n    }\n  }\n}"
+}
```
Comment (szabosteve) [inline specification/autoscaling/put_autoscaling_policy/autoscalingApisPutAutoscalingPolicyRequestExample2.json:7]:
```suggestion
  "summary": "Creates an autoscaling policy.",
  "method_request": "PUT /_autoscaling/policy/my_autoscaling_policy",
  "description": "The API method and path for this request: `PUT /_autoscaling/policy/my_autoscaling_policy. It creates `my_autoscaling_policy` using the fixed autoscaling decider, applying to the set of nodes having (only) the `data_hot` role.",
  "type": "request",
  "value": "{\n  \"roles\" : [ \"data_hot\" ],\n  \"deciders\": {\n    \"fixed\": {\n    }\n  }\n}"
}
```

**Pull request merged**